### ISSUE TITLE
Is there any evidence that NEDNSProxyProvider works only on supervised devices?

### ISSUE THREAD

**Issue by lubbo** [pusateri/DNS-TLS#3]
Do you have any update on such restriction?
Thanks!

**Comment by pusateri** [comment]
The log files showing the restriction are included here:
[https://dnsdisco.com/iOS-dns-proxy-post.html](https://dnsdisco.com/iOS-dns-proxy-post.html)

Recent discussions make seem as though this isn't going to change. However, I think opportunistic DNS over TLS will be added in a future release.

**Comment by aalmenar** [comment]
Now there’s a way to do it:

https://developer.apple.com/wwdc20/10047

**Comment by pusateri** [comment]
Yes! I need to update the README that this is probably no longer needed.

**Comment by iambenmitchell** [comment]
Hi, I am trying to make an app like cloud flares 1.1.1.1 app but for my own DNS server. I would configure it in wifi settings but I need it to work over mobile data too. Thus, I want to create a VPN like app but it only forwards DNS. Do you know what I should do?
